src/utils/validations.js

```diff
@@ -1 +1,22 @@
+import { DNS1123_START_ERROR, DNS1123_END_ERROR, DNS1123_CONTAINS_ERROR, DNS1123_EMPTY_ERROR } from './strings';
+
 export const isPositiveNumber = value => value && value.match(/^[1-9]\d*$/);
+
+// DNS-1123 subdomain
+export const validateDNS1123SubdomainValue = value => {
+  if (!value) {
```

Comment:
what about maximal length? (253 for dns1123 but do you know limit for k8s name?)

Reply:
good catch, I will add the length check